File: pgm/ABC_Kotlin/ABC_426_A.kt
// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "

fun main() {
    val (X: String, Y: String) = readln().split(space)
    val OS: List<String> = listOf("Ocelot", "Serval", "Lynx")
    if (OS.indexOf(Y) <= OS.indexOf(X)) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}

File: pgm/ABC_R/ABC_426_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
Yes <- "Yes"
No <- "No"
blank <- ""
space <- " "
yn <- "\n"
one <- 1L
two <- 2L
X <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][one]
Y <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][two]
OS <- c("Ocelot", "Serval", "Lynx")
if (match(x = Y, table = OS) <= match(x = X, table = OS)) {
  Yes |>
    cat(yn, sep = blank)
} else {
  No |>
    cat(yn, sep = blank)
}
q("no")